src/components/vr_cooperation/test/include/mock_vr_proxy_listener.h

```diff
@@ -30,46 +30,20 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include "vr_cooperation/commands/request_from_hmi.h"
-#include "vr_cooperation/vr_module.h"
+#ifndef SRC_COMPONENTS_VR_COOPERATION_TEST_INCLUDE_MOCK_VR_PROXY_LISTENER_H_
+#define SRC_COMPONENTS_VR_COOPERATION_TEST_INCLUDE_MOCK_VR_PROXY_LISTENER_H_
 
-namespace vr_cooperation {
-
-namespace commands {
-
-CREATE_LOGGERPTR_GLOBAL(logger_, "VRCooperation")
-
-RequestFromHMI::RequestFromHMI(VRModule* parent,
-                               const vr_hmi_api::ServiceMessage& message)
-    : message_(message),
-      parent_(parent) {
-}
-
-RequestFromHMI::~RequestFromHMI() {
-}
+#include "gmock/gmock.h"
+#include "vr_cooperation/vr_proxy_listener.h"
 
-void RequestFromHMI::OnTimeout() {
-  LOG4CXX_AUTO_TRACE(logger_);
-}
-
-void RequestFromHMI::Run() {
-  LOG4CXX_AUTO_TRACE(logger_);
-  Execute();    // run child's logic
-}
-
-void RequestFromHMI::on_event() {
-  LOG4CXX_AUTO_TRACE(logger_);
-  OnEvent();  // run child's logic
-}
-
-void RequestFromHMI::SendRequest() {
-  LOG4CXX_AUTO_TRACE(logger_);
-}
+namespace vr_cooperation {
 
-void RequestFromHMI::SendResponse() {
-  LOG4CXX_AUTO_TRACE(logger_);
-}
+class MockVRProxyListener : public VRProxyListener {
+ public:
+  MOCK_METHOD1(OnReceived,
+      void(const vr_hmi_api::ServiceMessage& message));
+};
```

Comment:
VRProxyListener have been changed recently because of mistakes. Please compare your version of classes for communicating with HMI and master branch version.